lib/engine/part/path.rb

```diff
@@ -34,6 +34,10 @@ def select(paths)
         on.keys.select { |p| on[p] == 1 }
       end
 
+      def edge_set
```

Comment:
i don't think we should do this, it's enough to just have the edge object which is unique, this just creates a more inefficient representation of this

so if you just do path.edges, that returns all you need to know

Reply:
ah maybe you just want the exit number, in that case, i still wouldn't do it like this,

just check it manually in the route

hex_edges = paths.flat_map do |path|
path.exits.map { |edge| [hex, edge] }
}

Reply:
Good call- I was trying to look for something like this. Will fix up. :+1: